jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -86,6 +86,28 @@ wait_pull_request_merge() {
     fi
 }
 
+fetch_build_timestamp() {
+    get_build_state
+    local artifacts_src_build_number=$(basename $target_url)
```

Comment:
It's hard to know where those variables are coming from. We should consider improving this format later.